lib/attr_encrypted/adapters/active_record.rb

```diff
@@ -56,9 +56,11 @@ def attr_encrypted(*attrs)
             attribute attr if ::ActiveRecord::VERSION::STRING >= "5.1.0"
             options.merge! attr_encrypted_encrypted_attributes[attr]
 
-            define_method("#{attr}_was") do
-              attribute_was(attr)
-            end
+            # attribute_was does not work with the way attribute_will_change! is being used
+            #
+            #define_method("#{attr}_was") do
+            #  attribute_was(attr)
+            #end
```

Comment:
I don't think you can remove this - If others were using this branch and dependent on this function it will now be gone